CODEOWNERS

```diff
@@ -0,0 +1,125 @@
+#   @jonasschnelli
+#   @marcofalke
+#   @meshcollider
+
+# Docs
+/doc/*[a-zA-Z-].md                          @harding
+/doc/Doxyfile.in                            @fanquake
+/doc/REST-interface.md                      @jonasschnelli
+/doc/benchmarking.md                        @ariard
+/doc/bitcoin-conf.md                        @hebasto
+/doc/build-freebsd.md                       @fanquake
+/doc/build-netbsd.md                        @fanquake
+/doc/build-openbsd.md                       @laanwj
+/doc/build-osx.md                           @fanquake
+/doc/build-unix.md                          @laanwj
+/doc/build-windows.md                       @sipsorcery
+/doc/dependencies.md                        @fanquake
+/doc/developer-notes.md                     @laanwj
+/doc/files.md                               @hebasto
+/doc/gitian-building.md                     @laanwj
+/doc/reduce-memory.md                       @fanquake
+/doc/reduce-traffic.md                      @jonasschnelli
+/doc/release-process.md                     @laanwj
+/doc/translation_strings_policy.md          @laanwj
+
+# Build aux
+/build-aux/m4/bitcoin_qt.m4                 @hebasto
+
+# MSVC build system
+/build_msvc/                                @sipsorcery
+
+# Settings
+/src/util/settings.*                        @ryanofsky
+
+# Fuzzing
+/src/test/fuzz/                             @practicalswift
+/doc/fuzzing.md                             @practicalswift
+
+# Test framework
+/test/functional/mempool_updatefromblock.py @hebasto
+/test/functional/feature_asmap.py           @jonatack
+/test/functional/interface_bitcoin_cli.py   @jonatack
+/test/functional/tool_wallet.py             @jonatack
+
+# Translations
+/src/util/translation.h                     @hebasto
+
+# Dev Tools
+/contrib/devtools/security-check.py         @fanquake
+/contrib/devtools/test-security-check.py    @fanquake
+/contrib/devtools/symbol-check.py           @fanquake
+
+# Gitian/Guix
+/contrib/gitian-build.py                    @hebasto
+/contrib/guix/                              @dongcarl
+
+# Compatibility
+/src/compat/glibc_*                         @fanquake
+
+# GUI
+/src/qt/forms/                              @hebasto
+
+# Wallet
+/src/wallet/                                @achow101
+
+# CLI
+/src/bitcoin-cli.cpp                        @jonatack
+
+# Coinstats
+/src/node/coinstats.*                       @fjahr
+
+# Index
+/src/index/                                 @fjahr
+
+# Descriptors
+*descriptor*                                @achow101 @sipa
+
+# Interfaces
+/src/interfaces/                            @ryanofsky
+
+# Scripts/Linter
+*.sh                                        @practicalswift
+/test/lint/                                 @practicalswift
+/test/lint/lint-shell.sh                    @hebasto
+/ci/lint/                                   @practicalswift
+
+# Bech32
+/src/bech32.*                               @sipa
+/src/bench/bech32.*                         @sipa
+
```

Comment:
Section for P2P handling? `/src/net_processing.*` and `/src/protocol.*`. Feel free to add me.